### DISCUSSION TITLE
alist上传成功，但是目录列表不会刷新出文件，需要重启服务才可以，但是网盘里面是可以看到alist上传的文件的

### DISCUSSION THREAD

**Discussion by wchangc** [AlistGo/alist#3726] [General]
alist上传成功，但是目录列表不会刷新出文件，需要重启服务才可以，但是网盘里面是可以看到alist上传的文件的
![image](https://user-images.githubusercontent.com/48704286/222971962-e568d017-f9ce-4a17-ac78-5789fa47a82d.png)
![image](https://user-images.githubusercontent.com/48704286/222971977-b4f8a32b-4bea-420c-b263-dd6b2ed2252f.png)

**Comment by anwen-anyi** [comment]
右下角刷新

**Reply by mashirozx** [reply]
为什么不做成自动刷新的呢。。

**Reply by liwanggui** [reply]
同样的问题，上传到 OSS 存储桶，文件显示不出来，需要重启才行，刷新也不顶用。还有其他方法解决吗？ @anwen-anyi

**Reply by wchangc** [reply]
尝试换一下其他版本试试看
